keyboards/ymdk/ymd62/config.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 Mike Tsao
```

Comment:
This PR is part my work, with some bits taken from the source code that YMDK shared with me for the YMD62 on their Discord.

I reversed the matrix using a multi-meter, and figured out the layout, added a keymap with keycodes for the RGB adjustments.

The bits I took from the YMDK code were the `#define`s for the LEDs, which I ported into `info.json`. The YMDK code was a bit weird - it seemed like it was for an ortho 75%. But the LED bits matched the YMD62.

If you want to see the source code that YMDK shared with me, you can get it via Discord. They have a link to their Discord on their website https://ymdkey.com/ , which links to this invitation: https://discord.com/invite/rvgWYRAacc . You can then see the RAR file that they shared with me at https://discord.com/channels/599111980590170132/784601964188139530/1212682121412608072 . Or I can upload it somewhere else for the QMK maintainers to review.